src/components/Nav.jsx

```diff
@@ -5,30 +5,36 @@ import navigationCSS from './css/navigation.module.css'
 const Nav = () =>{
 
     const [order, setOrder] = useState('desc')
+    const [sortBy, setSortBy] = useState('votes')
 
     function onChangeOrder(event){
         setOrder(event.target.value)
     }
+    function onChangeSortBy(event){
+        setSortBy(event.target.value)
+    }
 
     return(
         <>
         <div className={`${navigationCSS.navigation} ${navigationCSS.grid4}`}>
             <Link to="/" className={navigationCSS.link}>All</Link>
-            <Link to={`/reviews/hidden-roles`} className={navigationCSS.link}>hidden-roles</Link>
-            <Link to="/reviews/dexterity" className={navigationCSS.link}>dexterity</Link>
-            <Link to="/reviews/strategy" className={navigationCSS.link}>strategy</Link>
-            <Link to="/reviews/push-your-luck" className={navigationCSS.link}>push-your-luck</Link>
-            <Link to="/reviews/roll-and-write" className={navigationCSS.link}>roll-and-write</Link>
-            <Link to="/reviews/deck-building" className={navigationCSS.link}>deck-building</Link>
-            <Link to="/reviews/engine-building" className={navigationCSS.link}>engine-building</Link>
+            <Link to={`/reviews?category=hidden-roles&order=${order}&sort_by=${sortBy}`} className={navigationCSS.link}>hidden-roles</Link>
```

Comment:
Rather than hardcoding the query string, can you use setSearchParams to instead change the URL from within the component?